### ISSUE TITLE
Question about performance

### ISSUE THREAD

**Issue by baehaeli** [hongli-ma/RNAsmol#7]
Hello, thank you for releasing the RNAsmol code.

I trained the model using the repository without changing the default settings. The results I obtained are below:

<img width="2015" height="576" alt="Image" src="https://github.com/user-attachments/assets/4cc642ea-49d6-4d16-a591-47a3d5dff334" />

From these results, the performance on the ROBIN dataset looks extremely high, but the performance on the PDB dataset is very low.

Could you please confirm whether this level of performance is expected for the no augmentation condition?
I would like to know if the released code is intended to produce these results, or if there might have been a mistake on my side during training.

Any clarification would be very helpful. Thank you!

**Comment by hongli-ma** [comment]
Thank you for your feedback, could you please provide the loss curve from your training? It looks like the network might be overfitting.

**Comment by baehaeli** [comment]
Here is the training log as requested.
I have included all perturbation settings for both PDB and ROBIN.

[pdb_netperturbation.log](https://github.com/user-attachments/files/22427603/pdb_netperturbation.log)
[pdb_rnaperturbation.log](https://github.com/user-attachments/files/22427601/pdb_rnaperturbation.log)
[pdb_molperturbation.log](https://github.com/user-attachments/files/22427602/pdb_molperturbation.log)

[robin_molperturbation.log](https://github.com/user-attachments/files/22427610/robin_molperturbation.log)
[robin_netperturbation.log](https://github.com/user-attachments/files/22427612/robin_netperturbation.log)
[robin_rnaperturbation.log](https://github.com/user-attachments/files/22427611/robin_rnaperturbation.log)

**Comment by hongli-ma** [comment]
From the log file you provided, it appears that the training loss on the PDB dataset is not decreasing. Try increasing the learning rate during training and see if that helps.

**Comment by baehaeli** [comment]
Thank you for the suggestion. 
Could you please let me know the exact learning rate that was used in the paper for training RNAsmol?
I could not find this detail in repository or the paper, and I think having this information would be very helpful for reproducing the reported results.
